code/graphics/opengl/gropengltexture.cpp

```diff
@@ -1627,7 +1627,8 @@ void opengl_kill_render_target(bitmap_slot* slot)
 		fbo->renderbuffer_id = 0;
 	}
 
-	opengl_free_fbo_slot(fbo->fbo_id);
+	if(remove_slot)
```

Comment:
What exactly is this supposed to do? At this point the OpenGL framebuffer object is already destroyed so the only thing this controls is whether we free the slot in our array.

Reply:
I am not 100% sure why this is, but removing the object from our array will result in the Image not being renderable anymore (will just be transparent), If you leave this object, it's fine.

Reply:
Not sure why this would cause rendering to fail. However, since not calling this function effectively produces a memory leak it will need to be fixed before this is merged.